customizations/README.md

```diff
@@ -192,11 +192,11 @@ There are two alternatives when connecting the environment with an external post
     ```
     > Note: One kubernetes secret is going to be created automatically during the installation process with the `externalDatabase.password` and `externalDatabase.adminPassword` that you set in previous lines.
```

Comment:
You haven't changed the comment ⬆️ 

Maybe a `git grep externalDatabase ` could help 👁️

Reply:
Oh, this has been when I merge the latest changes of main, this Readme has been modified in https://github.com/CartoDB/carto-selfhosted-helm/pull/68